templates/components/products/price-range.html

```diff
@@ -1,21 +1,44 @@
-{{#and price.retail_price_range.min.without_tax price.retail_price_range.max.without_tax}}
-    <div class="price-section price-section--withTax" {{#if schema_org}}itemprop="offers" itemscope itemtype="http://schema.org/Offer"{{/if}}>
-        <span data-product-price-without-tax class="price price--rrp">{{price.retail_price_range.min.without_tax.formatted}} - {{price.retail_price_range.max.without_tax.formatted}}</span>
+{{#and price.retail_price_range.min.with_tax price.retail_price_range.max.with_tax}}
+    <div class="price-section price-section--withTax rrp-price--withTax" {{#if schema_org}}itemprop="offers" itemscope itemtype="http://schema.org/Offer"{{/if}}>
+        {{lang 'products.retail_price'}}
+        <span data-product-rrp-price-with-tax class="price price--rrp">{{price.retail_price_range.min.with_tax.formatted}} - {{price.retail_price_range.max.with_tax.formatted}}</span>
         {{#if schema_org}}
             <meta itemprop="availability" content="{{product.availability}}">
             <meta itemprop="itemCondition" itemtype="http://schema.org/OfferItemCondition" content="http://schema.org/{{product.condition}}Condition">
             <div itemprop="priceSpecification" itemscope itemtype="http://schema.org/PriceSpecification">
-                <meta itemprop="minPrice" content="{{price.retail_price_range.min.without_tax.value}}"  />
-                <meta itemprop="price" content="{{price.retail_price_range.min.without_tax.value}}">
-                <meta itemprop="maxPrice" content="{{price.retail_price_range.max.without_tax.value}}"  />
+                <meta itemprop="minPrice" content="{{price.retail_price_range.min.with_tax.value}}"  />
+                <meta itemprop="price" content="{{price.retail_price_range.min.with_tax.value}}">
+                <meta itemprop="maxPrice" content="{{price.retail_price_range.max.with_tax.value}}"  />
                 <meta itemprop="priceCurrency" content="{{currency_selector.active_currency_code}}">
                 <meta itemprop="valueAddedTaxIncluded" content="true">
             </div>
         {{/if}}
     </div>
+    {{else}}
+        {{#if price.with_tax}}
+            <div class="price-section price-section--withTax rrp-price--withTax" style="display: none;">
+            {{lang 'products.retail_price'}}
+            <span data-product-rrp-with-tax class="price price--rrp">
+                {{#if price.rrp_with_tax}}
```

Comment:
might want to have this if check rest above `{{lang 'products.retail_price'}}` that way it doesnt show the retail price text without there being a price to display.

Reply:
The retail price text will not show until JS updates this element to no longer be hidden.